### ISSUE TITLE
Eternal Crashing with latest update

### ISSUE THREAD

**Issue by yodeladywho** [avocode/avocode#2066]
I cannot open the desktop version with the latest update. It gets to the preview screen, doesn't load any previews and then crashes. I have downloaded the latest version directly from the site as well, same issues. All MacOS/Software is up to date.

---
* **Version**: 3.1.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
---
```
{}
```

**Comment by petrbrzek** [comment]
I'm sorry for the trouble. You can try manually remove `.avocode` folder. It's located in your home directory. It'll remove the local application state. That also means you will lose some local settings like theme or font-size. 
Let us know if it's working after that.

**Comment by yodeladywho** [comment]
nope:/ not sure what's up - we are 3/4 for computers in the office. 3 out of four not working I mean. 

I deleted the .avocode folder, removed any traces of the program, emptied my trash, rebooted and then reinstalled. Still the same issue. I get to the log in screen now, try to interact and it just crashes.

**Comment by petrbrzek** [comment]
@yodeladywho can you launch the app from a terminal like this `open -a /path-to-avocode/Avocode.app  --args -- --dev-tools`. It should launch the app with dev tools. Can you send us back what is inside the console?

**Comment by yodeladywho** [comment]
I grabbed what I could - 
```
/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/raven-js/src/console.js:35 AuthManager#initAuth() -> auth error: 401 Error: No token data
    at e.value (avocode.min.js:1)
    at t.value (avocode.min.js:1)
    at t.value (avocode.min.js:1)
    at e.value (avocode.min.js:1)
    at Object.<anonymous> (avocode.min.js:1)
    at i (avocode.min.js:1)
    at Object.<anonymous> (avocode.min.js:1)
    at i (avocode.min.js:1)
    at avocode.min.js:1
    at avocode.min.js:1
console.(anonymous function) @ /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/raven-js/src/console.js:35
/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/raven-js/src/console.js:35 (node:1509) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SWAP listeners added. Use emitter.setMaxListeners() to increase limit
console.(anonymous function) @ /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/raven-js/src/console.js:35
/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/raven-js/src/console.js:35 (node:1509) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 change listeners added. Use emitter.setMaxListeners() to increase limit
console.(anonymous function) @ /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/raven-js/src/console.js:35
/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/rave…:35 AuthStateIo#invalidate() -> could not delete .token file: Error: ENOENT: no such file or directory, unlink '/Users/chrislacey/.avocode/.token'
console.(anonymous function) @ /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/stage/node_modules/rave…:35
(anonymous) @ avocode.min.js:1
(anonymous) @ fs.js:114
frame.ac8bfe1a.js:1 POST https://api-iam.intercom.io/messenger/web/ping 400 (Bad Request)
(anonymous) @ frame.ac8bfe1a.js:1
o @ frame.ac8bfe1a.js:1
(anonymous) @ frame.ac8bfe1a.js:1
e.exports.post @ frame.ac8bfe1a.js:1
o @ frame.ac8bfe1a.js:1
o @ frame.ac8bfe1a.js:1
(anonymous) @ frame.ac8bfe1a.js:1
(anonymous) @ frame.ac8bfe1a.js:1
i @ frame.ac8bfe1a.js:1
```

**Comment by yodeladywho** [comment]
build from #2068 seems to have worked for me as well.

**Comment by jakubzitny** [comment]
We released `3.1.1` with the same fix as in the build above, it should work properly. Closing for now.
